firmware/validation/sil/sil_client.h

```diff
@@ -0,0 +1,38 @@
+/// @author Samuel Parent
+/// @date 2024-05-01
+
+#pragma once
+
+#include <grpcpp/grpcpp.h>
+
+#include <iostream>
+#include <memory>
+#include <string>
+
+#include "grpcpp/channel.h"
+#include "signals.grpc.pb.h"
+#include "signals.pb.h"
+
+namespace val::sil {
+
+class SilClient {
+public:
+    SilClient(std::string server_addr);
+
+    void Connect();
+
+    void RegisterDigitalInput(std::string ecu_name, std::string sig_name);
```

Comment:
this style will work for now, but this repeated `Register___()` may be solved with OOP on the different peripherals